R/utils.R

```diff
@@ -145,12 +145,16 @@ quote_wrap <- function(x, q) paste0(q, x, q)
 # interface to work like options() or setwd() -- returns the old value for convenience
 set_lang <- function(new_lang) {
   old_lang <- Sys.getenv("LANGUAGE", unset = NA)
-  Sys.setenv(LANGUAGE = new_lang)
+  Sys.setenv(LANGUAGE = new_lang) # nolint: undesirable_function_linter.
   old_lang
 }
 # handle the logic of either unsetting if it was previously unset, or resetting
 reset_lang <- function(old_lang) {
-  if (is.na(old_lang)) Sys.unsetenv("LANGUAGE") else Sys.setenv(LANGUAGE = old_lang)
```

Comment:
Can't add comment inline because that will trigger `line_length()` linter.

Reply:
NB wondering why is `Sys.unsetenv()` not undesirable.
Maybe a follow up?

Reply:
SGTM.

Reply:
Created issue: https://github.com/r-lib/lintr/issues/1762